### DISCUSSION TITLE
What's `github.com/notifications/beta/archive`?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [community/community#42537] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

When I access my notifications, when I invoke a notification, I am sometimes redirected to https://github.com/notifications/beta/archive. This only started occurring recently. Any idea of why?

**Comment by jfullstackdev** [comment]
that's not happening to me, can you have a screenshot of that ? 

that link is 404 on my end too

**Reply by roke-julian-lockhart** [reply]
It doesn't show a 404, rather, just a blank page. Navigating to it manually indeed shows a 404.

**Comment by JonnyTech** [comment, marked as answer]
This has started happening to me too, today.

- Go https://github.com/notifications/
- Click on the tick for an item to set it as `Done`
- Redirected to https://github.com/notifications/beta/archive

The page is blank, no source, no nothing.

From my browser developer tools:

```
Uncaught DOMException: Element.matches: '�js-form-toggle-target' is not a valid selector vendors-node_modules_github_remote-form_dist_index_js-node_modules_github_session-resume_dist-def857-2a32d97c93c5.js:1
This page is in Quirks Mode. Page layout may be impacted. For Standards Mode use “<!DOCTYPE html>”. archive
```

Incidentally, the notification is correctly marked as done.

EDIT: Looks UBlock Origin is blocking ​https://api.github.com/_private/browser/stats

**Reply by roke-julian-lockhart** [reply]
I'm using uBlock Origin too.

**Comment by JonnyTech** [comment]
UPDATE:
- uninstalled ublock origin (no other extensions installed) - issue still occurs
- tried with chromium - issue does not happen

So, it's something with Firefox. Any suggestions?

**Reply by roke-julian-lockhart** [reply]
I'm using Firefox too.

**Comment by JonnyTech** [comment]
It "fixed itself" after a few hours...

**Reply by roke-julian-lockhart** [reply]
It'll reappear. It always does.

**Comment by CoolCat467** [comment]
I've seen this too from time to time. I predict that the javascript frontend is dynamically replacing something, and that if you navigate too quickly it's left in its original, un-modified form.
